input/data/2011/2011ApJ...738....3A/tev-000014-lc.ecsv

```diff
@@ -0,0 +1,65 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: significance, datatype: float32}
+# - {name: flux, unit: ph / (cm2 s), datatype: float64}
+# - {name: flux_err, unit: ph / (cm2 s), datatype: float64}
+# - {name: flux_ul, unit: ph / (cm2 s), datatype: float64}
+# meta: !!omap
+# - data_type: lc
+# - source_id: 14
+# - source_name: LS I +61 303
```

Comment:
We've decided to remove `source_name` keys from data input files, and to only keep `source_id`.
So please remove it here.

The reasoning (see https://github.com/gammapy/gamma-cat/issues/86) was that it's extra work to check two pieced of info that contain the same info.

While I'm doing data entry, I'm using
https://gammapy.github.io/gamma-cat/sources.html
to look up `source_id` from names.